main_script.py

```diff
@@ -0,0 +1,113 @@
+from data_processing_scripts.dna_rna_tools import transcribe, reverse, complement, reverse_complement
+from data_processing_scripts.das_protein_tools import get_pI, calculate_aa_freq, translate_protein_rna, convert_to_3L_code, protein_mass
+from data_processing_scripts.fastq_script import main_fastq_tools, parse_file, save_filtered_fastq
+import data_processing_scripts.dna_rna_dict as drd
+import data_processing_scripts.protein_dict as prd
+import os
+
+
+def main_dna_rna_tools(*args: str):
```

Comment:
1) Функции должны называться глаголами
2) Ну че ж тут как то как будто мы студенты:))
```suggestion
def run_dna_rna_tools(*args: str):
```